automation/render-local-env.sh

```diff
@@ -1,23 +1,82 @@
 #!/usr/bin/env bash
+set -e
+
+## Run from automation/
+## Clones the firecloud-automated-testing repo, pulls templatized configs, and renders them to src/test/resources
 
 # Defaults
 WORKING_DIR=$PWD
 VAULT_TOKEN=$(cat ~/.vault-token)
+FIRECLOUD_AUTOMATED_TESTING_BRANCH=master
 ENV=dev
-LOCAL_UI=false
+SERVICE=rawls
+LOCAL_UI=${LOCAL_UI:-false}  # local ui defaults to false unless set in the env
 
 # Parameters
-WORKING_DIR=${1:-$WORKING_DIR}
-VAULT_TOKEN=${2:-$VAULT_TOKEN}
-ENV=${3:-$ENV}
-if [ "$4" = "local_ui" ]; then
-  LOCAL_UI=true 
-fi
-
-
-# render ctmpls
-docker pull broadinstitute/dsde-toolbox:dev
-docker run --rm -e VAULT_TOKEN=${VAULT_TOKEN} \
-    -e ENVIRONMENT=${ENV} -e ROOT_DIR=${WORKING_DIR} -v ${WORKING_DIR}:/working \
-    -e OUT_PATH=/working/src/test/resources -e INPUT_PATH=/working/docker -e LOCAL_UI=$LOCAL_UI \
-    broadinstitute/dsde-toolbox:dev render-templates.sh
+FIRECLOUD_AUTOMATED_TESTING_BRANCH=${1:-$FIRECLOUD_AUTOMATED_TESTING_BRANCH}
+WORKING_DIR=${2:-$WORKING_DIR}
+VAULT_TOKEN=${3:-$VAULT_TOKEN}
+ENV=${4:-$ENV}
+
+confirm () {
+    # call with a prompt string or use a default
+    read -r -p "${1:-Are you sure?} [y/N] " response
+    case $response in
+        [yY])
+            shift
+            $@
+            ;;
+        *)
+            ;;
+    esac
+}
+
+# clone the firecloud-automated-testing repo
+clone_repo() {
```

Comment:
Maybe not for today, but I was thinking about this over the weekend... couldn't we just bundle the scripts and templates into a docker image instead of cloning the repo?